frontend/package.json

```diff
@@ -58,7 +58,7 @@
     "lint-staged": "10.0.8",
     "prettier": "1.19.1",
     "react-test-renderer": "16.12.0",
-    "redux-mock-store": "1.5.4"
+    "redux-mock-store": "^1.5.4"
```

Comment:
Hi, redux-mock-store latest version is already 1.5.4 (as of today) so why would put the "^" in front of it? As we saw with some of our dependencies, they usually break the Docker build (npm install) when they are not locked down to exact versions. 

By freezing these dependencies we want to achieve repeatable deployments and make sure that every developer is testing on the very same codebase.

For updating dependencies, there are a few free tools out there that we could explore to automate the update process. I like how [renovate](https://github.com/renovatebot/renovate) or [dependabot](https://dependabot.com/#how-it-works) (acquired by github) issue PRs with the updates and gives the team a chance to review whenever there's a new dependency that needs updating.

Reply:
Yes, adding the ^ was not intentional so using one of those tools might help.